test/transform/CompatibilityTransform.test.js

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert'
+import domino from 'domino'
+import pagelib from '../../build/wikimedia-page-library-transform'
+
+describe('CompatibilityTransform', () => {
+  it('.enableSupport()', () => {
+    const document = domino.createDocument()
+    pagelib.CompatibilityTransform.enableSupport(document)
+    for (const clazz of Object.values(pagelib.CompatibilityTransform.COMPATIBILITY)) {
```

Comment:
I find `thisClass` to be a little distracting because what I'm really wanting is a simple singular of a plural, classes. I'm kind of anti-`myVariable` / `thisVariable` :] I didn't use `className` to avoid confusion with `Element.className`. I wouldn't mind `value` or `classValue` if you prefer it.